docs/faq.rst

```diff
@@ -0,0 +1,46 @@
+Frequently Asked Questions
+==========================
+
+What is SMQTK?
+^^^^^^^^^^^^^^
+SMQTK is an open-source light-weight framework for developing interfaces that have built-in implementation discovery and factory construction from configuration.
+
+Why would I use SMQTK over KWIVER?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You should use SMQTK if you want to define your own python algorith implementations and don't want to develop in C++. However, if the algorithm implenetations that you want to use are already defined in KWIVER, then KWIVER would be the better option.
+
+I've used SMQTK before, but what are these broken out packages?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In 2021, ``SMQTK v0.14.0`` was broken out from one monolothic library into several distinct libraries labeled as ``SMQTK-Core``, ``SMQTK-Classifier``, ``SMQTK-Image-IO``, and more. The decision was part of a new effort to reduce technical debt and isolate functionality to preserve the light-weight design.
+
+Can I contribute to SMQTK?
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Of course! To add in your own implementation see ``CONTRIBUTING.md``. Additionally you can contribute by helping review any outstanding branches in any one of the SMQTK repos. For guidelines on reviewing please see `review_process`_
+
+.. _`review_process`: review_process.html
+
+What does SMQTK encompass?
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+SMQTK is currently composed of 7 different libraries, all of which are pip installable:
```

Comment:
I think we want to communicate that SMQTK[-Core] is not *just* these packages, but that this package is a "core" that these other packages are examples that utilize this, w.r.t. using the `Pluggable` and `Configurable` mixins to build out more complex functionality and provide pluggable implementations.

Reply:
I updated the answer to this question. Let me know if you think its clear enough